docs/source/distributions/building_distro.md

```diff
@@ -178,7 +178,7 @@ image_name: ollama
 image_type: conda
 
 # If some providers are external, you can specify the path to the implementation
-external_providers_dir: /etc/llama-stack/providers.d
+external_providers_dir: ~/.llama/providers.d
```

Comment:
Let's clarify that any directory will work - I used `/etc/llama-stack/providers.d` because that's the Linux path for daemon config files. I'm not opposed to offer `~/.llama/providers.d` as an example though.

Reply:
yeah, being able to point to any dir makes sense! With this change though, we have a default such that external providers can be read if found in ~/.llama/providers.d

if we don't want this behavior I can close this, but it would make it easier for external providers to conform around ~/.llama/providers.d

Reply:
No I think it's important we converge to a single location so that external provider packages can put their config file in a know directory.

Reply:
can we standardize around `~/.llama/providers.d`? this will allow external provider packages to write their adapter YAMLs directly to where LLS expects them without needing root access

Reply:
@nathan-weinberg yes!